Sources/TuistKit/Commands/Project/ProjectUpdateCommand.swift

```diff
@@ -21,6 +21,11 @@ struct ProjectUpdateCommand: AsyncParsableCommand {
     )
     var defaultBranch: String?
 
+    @Option(
+        help: "Set the connected git repository. Example: --repository https://github.com/tuist/tuist"
```

Comment:
```suggestion
        help: "Set the connected Git repository. Example: --repository https://github.com/tuist/tuist"
```

What happens if I pass `https://github.com/tuist/tuist.git`?

Reply:
That's also valid.